scripts/backup.js

```diff
@@ -95,6 +95,10 @@ function getDataFileNames() {
     return ['kademlia.crt', 'kademlia.key', 'houston.txt', 'system.db'];
 }
 
+function getMigrationFileNames() {
```

Comment:
Can we read file names from migrations folder instead of adding them here by names? There is a great possibility that we will forget to add new migrations in the future. Something like:

```
const` fs = require('fs');
fs.readdir(directoryPath, function (err, files) {
    files.forEach(function (file) {
        ...
    });
});
```
Same thing for restore.js

Reply:
Good point!